src/cmd/cmd.cpp

```diff
@@ -352,7 +351,7 @@ int main(int argc, char **argv)
         return EXIT_FAILURE;
     }
 
-    // check if the dav path was added to the url and append if not.
+    // append trailing slash (not sure if it is still needed)
```

Comment:
maybe remove that comment or remove that if statement altogether?

Reply:
at least all examples do not have the trailing `/`
so would need to check if it is really needed and if yes update the examples to have the trailing `/`